asar-teensy-firmware/TestScripts/Motors/TestScript1/TestScript1.ino

```diff
@@ -18,21 +18,7 @@ void setup()
 void loop()
 {
   myMOTOR.Forward(5);
-  delay(2000);
```

Comment:
Why change the motor test?

Reply:
I have been messing with these tests to show different things, I should finalize and keep them constant thought